pkg/ebpf/utils_linux.go

```diff
@@ -3,14 +3,88 @@
 package ebpf
 
 import (
+	"bufio"
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
Can you run `goimports` on this file?

Reply:
Hey @brycekahle, no imports updated when I ran this.